src/libraries/System.Reflection.MetadataLoadContext/tests/System.Reflection.MetadataLoadContext.Tests.csproj

```diff
@@ -86,7 +86,7 @@
     <Compile Include="$(CommonTestPath)System\ModifiedTypeTests.cs" Link="Common\System\ModifiedTypeTests.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetOS)' == 'browser'">
+  <ItemGroup Condition="'$(TargetOS)' == 'browser' or ('$(EnableAggressiveTrimming)' == 'true' and '$(UseNativeAotRuntime)' != 'true')">
```

Comment:
nit:

We should split this into two ItemGroups:
- one setting up `WasmFilesToIncludeFromPublishDir` only when `Condition="'$(TargetOS)' == 'browser'"`
- and another one which sets `TrimmerRootDescriptor` when `Condition="'$(TargetOS)' == 'browser' or ('$(EnableAggressiveTrimming)' == 'true' and '$(UseNativeAotRuntime)' != 'true')"`

otherwise we would set `WasmFilesToIncludeFromPublishDir` even if it is not used.

Reply:
Fixed